### PR TITLE
FileSystem: add functions

### PR DESCRIPTION
**Pull request by cynepco3hahue**
- mkdir
- chown
- chmod

### CODE REVIEW

File: rrmngmnt/filesystem.py [hunk 1]
```diff
@@ -54,3 +55,51 @@ def touch(self, file_name, path):
...
+    def mkdir(self, path):
+        """
+        Create directory on host
+
+        :param path: directory path
+        :type path: str
+        :return: True, if action succeed, otherwise False
+        :rtype: bool
+        """
+        return self.host.executor().run_cmd(
```
Comment (myakove) [inline rrmngmnt/filesystem.py]: run_command

File: rrmngmnt/filesystem.py [hunk 2]
```diff
@@ -54,3 +55,51 @@ def touch(self, file_name, path):
...
+        """
+        Change permission of directory or file
+
+        :param path: file or directory path
+        :type path: str
+        :param mode: permission mode(600 for example)
+        :type mode: str
+        :return: True, if action succeed, otherwise False
+        :rtype: bool
+        """
```
Comment (myakove) [inline rrmngmnt/filesystem.py]: Do you want to check of mode is numbers only?

File: rrmngmnt/filesystem.py [hunk 3]
```diff
@@ -54,3 +55,51 @@ def touch(self, file_name, path):
...
+        :type path: str
+        :param username: change user owner to username
+        :type username: str
+        :param groupname: change group owner to groupname
+        :type groupname: str
+        :return: True, if action succeed, otherwise False
+        :rtype: bool
+        """
+        uid, gid = (
+            pwd.getpwnam(owner).pw_uid for owner in (username, groupname)
```
Comment (lukas-bednar) [inline rrmngmnt/filesystem.py]: what if I want to manipulate, user which doesn't exist on local machine, but it does on remote machine ...

**Comment by lukas-bednar** [comment]
I am ok with this patch, please get rid of pwd module since it works with local user database, and doesn't reflect what users are on remote machine ...

**Comment by cynepco3hahue** [comment]
Will create new one without rebase

**Pull request closed without merging**